vhdbuilder/packer/install-dependencies.sh

```diff
@@ -153,6 +153,9 @@ if [[ ${CONTAINER_RUNTIME:-""} == "containerd" ]]; then
     downloadCrictl ${CRICTL_VERSION}
     echo "  - crictl version ${CRICTL_VERSION}" >> ${VHD_LOGS_FILEPATH}
   done
+  
+  KUBERNETES_VERSION=$(echo -e "$CRICTL_VERSIONS" | head -n 1) installCrictl || exit $ERR_CRICTL_DOWNLOAD_TIMEOUT
```

Comment:
Is the output stored in KUBERNETES_VERSION, being used elsewhere?

Reply:
in installCrictl :)